x-pack/plugins/alerting/server/rules_client_factory.ts

```diff
@@ -52,6 +52,7 @@ export interface RulesClientFactoryOpts {
   alertsService: AlertsService | null;
   connectorAdapterRegistry: ConnectorAdapterRegistry;
   uiSettings: CoreStart['uiSettings'];
+  maxAlertsPerRun: ActionsConfig['max'];
```

Comment:
I'll let the ResponseOps team check this, but it looks this should be `AlertingConfig['rules']['run']['alerts']['max']`,

or create that type in `x-pack/plugins/alerting/server/config.ts` and export it.

Reply:
nit:

If you extend `AlertRulesConfig` like so:
```ts
export type AlertingRulesConfig = Pick<
  AlertingConfig['rules'],
  'minimumScheduleInterval' | 'maxScheduledPerMinute' | 'run'
> & {
  isUsingSecurity: boolean;
};
```

You can then use here:
```ts
maxAlertsPerRun: AlertingRulesConfig['run']['alerts']['max'];
```